test/cpp/interop/xds_interop_client.cc

```diff
@@ -55,6 +55,8 @@ ABSL_FLAG(int32_t, stats_port, 50052,
 ABSL_FLAG(std::string, rpc, "UnaryCall",
           "a comma separated list of rpc methods.");
 ABSL_FLAG(std::string, metadata, "", "metadata to send with the RPC.");
+ABSL_FLAG(std::string, expect_status, "OK",
```

Comment:
I believe the absl flag API provides the ability to register a verifier function to be run when a value is specified for a flag.  We should use that to ensure at startup time that we are not given an invalid value here.